frontend_app_gallery/src/component/MainPreview.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+class MainPreview extends React.Component {
+  onLoad ({ target: img }) {
+    this.height = img.height
```

Comment:
I will need to know why width and height are class property instead of state. I'm not saying it's wrong, I just need an explanation

Reply:
It works but it is wrong, because `onLoad` will be called after render the component so it need to update with knowing the height and width, but in this case the img load with `rotationAngle` = 0 so the width doesn't need to be set.

But for better constancy, `height` and `width` will be in the state